### PR TITLE
fix(VAutocomplete): show selections slot and the input side-by-side like v2 does

### LINKED ISSUE #15756: [Bug Report][3.0.0-beta.9] v-autocomplete - selection slot isn't rendering right after selecting item
### Environment
**Vuetify Version:** 3.0.0-beta.9
**Vue Version:** 3.2.34
**Browsers:** Chrome 105.0.0.0
**OS:** Linux x86_64

### Steps to reproduce
select item from v-autocomplete component

### Expected Behavior
selection slot is rendered

### Actual Behavior
item-title is rendered

### Reproduction Link
<a href="https://codepen.io/kony97/pen/xxjwWQo" target="_blank">https://codepen.io/kony97/pen/xxjwWQo</a>

### PR DESCRIPTION
**Pull request by yuwu9145**
fixes #15756

## Description

fixes #15756

## How Has This Been Tested?

visually

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
<template>
  <v-app>
    <v-container>
      <v-autocomplete :items="countries" item-title="name" item-value="code" >
        <template #selection="{ item }">
          {{ item.raw.code }} {{ item.raw.name }}
        </template>
        <template #item="{ props, item }">
          <v-list-item v-bind="props" title="">
          {{ item.raw.code }} {{ item.raw.name }}
          </v-list-item>
        </template>
      </v-autocomplete>

      <v-autocomplete
        :items="items"
        dense
        small-chips
        label="Solo"
        solo
      ></v-autocomplete>
      
      <h1>Select</h1>      
      <v-select :items="countries" item-title="name" item-value="code">
        <template #selection="{ item }">
          {{ item.raw.code }} {{ item.raw.name }}
        </template>
        <template #item="{ props, item }">
          <v-list-item v-bind="props" title="">
          {{ item.raw.code }} {{ item.raw.name }}
          </v-list-item>
        </template>
      </v-select>
      
      <h1>combobox</h1>      
      <v-combobox multiple :items="items">
        <template #selection="data">
          {{ data.item.title }}
        </template>
        <template #item="{props, item }">
          <v-list-item v-bind="props">
          </v-list-item>
        </template>
      </v-combobox>

      <v-combobox
        :items="items"
        dense
        small-chips
        multiple
        label="Solo"
        solo
      ></v-combobox>

    </v-container>
  </v-app>
</template>

<script>
  export default {
    name: 'Playground',
    setup () {
      return {
        //
      }
    },
    methods: {
      alertListener() {
        alert('Hello')
      }
    },
    data: () => ({
      countries: [
        {
          name: "AAA",
          code: "1",
        },
        {
          name: "BBB",
          code: "2",
        },
        {
          name: "CCC",
          code: "3",
        },
        {
          name: "DDD",
          code: "4",
        },
      ],
      items: ['foo', 'bar', 'fizz', 'buzz'],
      values: ['foo', 'bar'],
      value: null,
  }),
  }
</script>

```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by KaelWD** [comment]
This causes #14009 again. We need to show selections and the input side-by-side like v2 does.

**Comment by yuwu9145** [comment]
Thanks @KaelWD , I tried that "side-by-side" already, just I wasn't sure whether that was the right solution so I pushed a rather simple fix in this PR.

I can try it again so leave it to me.

**Comment by yuwu9145** [comment]
@KaelWD changes have been made to cover:

- For plain text selection (no slot): input field displays as absolute so it can cover selected text
- For selection slot is applied: display selection slot and the input side-by-side like v2 does

File: packages/vuetify/src/components/VAutocomplete/VAutocomplete.sass [hunk 1]
```diff
@@ -85,5 +85,10 @@
         opacity: 0
 
     .v-field--focused
-      .v-autocomplete__selection
+      .v-autocomplete__selection-text
         opacity: 0
+
+  &--slection-slot
```
Comment (johnleider) [inline packages/vuetify/src/components/VAutocomplete/VAutocomplete.sass:91]:
```suggestion
  &--selection-slot
```

**Review by johnleider — CHANGES_REQUESTED** [review]
This should also apply to VSelect and VCombobox.

**Comment by yuwu9145** [comment]
Hi @johnleider 

Maybe I missed something, but as per my understanding at this stage. It does not seem to be necessary to apply this effect to `VSelect` and `VCombobox`:
- The root cause of this issue is `input` tag sometimes show over selection slot when there was manual user input, so input value looks overrides selection slot
- For VSelect, its `input` tag is always hidden when selected and VSelect should be designed to disallow any manual user input. Therefore, `input` tag cause no issue here.
- For VCombobox, its current behaviour looks fine.Its input tag always displays after selected items/slots. So also no issue in this senario.

I have updated sample markup to include `VCombobox` and `VSelect`. Feel free to leave further feedback :)

**Review by johnleider — APPROVED** [review]

**Pull request merged**